bear.profile

```diff
@@ -45,4 +48,14 @@ function bear_form_node_form_alter(&$form, FormStateInterface $form_state) {
   if (isset($form['sticky'])) {
     $form['sticky']['#access'] = FALSE;
   }
+
+  $user = \Drupal::currentUser();
+  if (\Drupal::moduleHandler()->moduleExists('workbench_moderation') && isset($form['moderation_state'])) {
```

Comment:
It has an alpha version, but we haven't included it in bear yet. I'll remove this stuff for now.